pkg/k8shandler/indexmanagement/index_management.go

```diff
@@ -0,0 +1,170 @@
+package indexmanagement
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	loggingv1alpha1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	esapi "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+)
+
+const (
+	PolicyNameApp   = "app-policy"
+	PolicyNameInfra = "infra-policy"
+	PolicyNameAudit = "audit-policy"
+
+	MappingNameApp   = "node.app"
```

Comment:
I should have linked in my changes based on our last team discussion: https://github.com/openshift/enhancements/blob/ae24f1c13487c5f02bab992219c520b783b6c5a2/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model

The mappingName = initial index so they should be: `app, infra, audit.infra` respectively